### ISSUE TITLE
Updating "expanded" prop using state. Works until you manually go back to the previous item. 

### ISSUE THREAD

**Issue by jasonpberry** [springload/react-accessible-accordion#176]
I created a demo of my issue using create react app.

Demo: https://jasonpberry.github.io/react-accessible-accordion/

I'm looking to use a [ Save & Continue ] button to open the next item  Vs manually clicking on the next item you wish to open.  All that is working great.

The issue I'm running into is when you click the [ Save & Continue button.  Then **manually** go back to the previous item you were on and click the button again.  The state updates.  The item opened no longer closes and opens the next item as it does the initial run through.

To show the state is updating I append the item title when the button click event is triggered to show that update.

Source: https://github.com/jasonpberry/react-accessible-accordion/tree/master/src

Am I trying to use this in a way that isn't supported?  

Any help is much appreciated.  The first issue I'm opening.  Apologize if I'm not providing as much detail as I should. 

Jason

**Comment by ryami333** [comment]
Gidday @jasonpberry, thanks for your issue. First of all, thanks for the demo and source you've provided, it really helps me to understand what your issue is.

> Am I trying to use this in a way that isn't supported?

_Essentially_, yes, but there's some buggy behaviour contributing to your experience here too. In the docs, you'll see that the `expanded` prop is only meant to `Expands this item on first render`. However, we've got some code in there which is forcing the `AccordionItem` to observe changes to the value of that prop too. Those two mechanisms (the internal one, and the prop-based one) fight against one another resulting in some nasty buggy behaviour.

So, in the upcoming release (see the `next` branch) we've removed the ability to control the expanded state manually. As justification, I recommend you check out my response to [this issue](https://github.com/springload/react-accessible-accordion/issues/174) and [the PR](https://github.com/springload/react-accessible-accordion/pull/131) it mentions.

The TL;DR is that if you need this kind of functionality, it _probably_ means that you're just implementing a component which is accordion-like in terms of look-and-feel, but not one which matches the WAI ARIA spec's definition of an 'Accordion'. These questions and use-cases keep on popping up though, so we probably need to do a better job of communicating in our README what this project is and isn't.

**Comment by jasonpberry** [comment]
@ryami333  - Appreciate the reply.  Totally understand how you're looking to stick to the WAI ARIA specs.  I'm probably looking for something along the lines of this:

https://www.npmjs.com/package/react-collapsible

**Comment by ryami333** [comment]
Yes, I would agree that this component is probably not suitable for your needs, but note that the component which you've linked to has zero concern for accessibility. Good luck with your implementation 😄

**Comment by jasonpberry** [comment]
Very true. Ugh.  Your solution is so perfect :) .  Just that one issue :)
